q/qt5-base/stone.yaml

```diff
@@ -4,11 +4,11 @@
 # SPDX-License-Identifier: MPL-2.0
 #
 name        : qt5-base
-version     : 5.15.16
-release     : 2
+version     : 5.15.16+kde.2529f7f0
```

Comment:
this should be `+git.$shortRef` - `kde` isn't a VCS its just the upstream we're pulling from. If the concern is confusion around the *origin* - perhaps we also need a way to incorporate this?